.github/CODEOWNERS

```diff
@@ -69,6 +69,7 @@ subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc
 subprojects/docs/src/docs/userguide/migration/migrating_from_groovy_to_kotlin_dsl.adoc  @eskatos @jbartok
 
 # Documentation
+subprojects/docs/src/docs/userguide/                                              @lkasso
```

Comment:
One of the quirky ways that CODEOWNERS works is that the last matching line wins, so you wouldn't get added to reviews in any of the userguide paths below. 

You'll have to add yourself explicitly to each of the lines below to get included.

Reply:
This has been updated in the latest commit. Thank you for the tip.